src/main/java/com/wire/xenon/backend/models/Payload.java

```diff
@@ -86,33 +83,14 @@ public static class Connection {
         @JsonProperty
         public String status;
 
-        @JsonProperty
-        public UUID from;
+        @JsonProperty("qualified_from")
+        public QualifiedId from;
 
-        @JsonProperty
-        public UUID to;
+        @JsonProperty("qualified_to")
+        public QualifiedId to;
 
-        @JsonProperty("conversation")
-        public UUID convId;
-    }
-
-    // User Mode
-    @JsonIgnoreProperties(ignoreUnknown = true)
-    public static class User {
```

Comment:
Is this class gone completely? I'm not able to follow all changes, perhaps it was supposed to, but it looks suspicious :)

Reply:
Yes, it was not needed, there is another `User` class used for API responses that can function for many different response types. So this static class is superfluous.